docs/guide/lesson-module.md

```diff
@@ -16,13 +16,13 @@ See the GIF below:
 
 ![Creating a module](img/addressbook-create.gif "Creating module with commands")
 
-You'll notice the created file structure in the ```modules/addressbook``` directory:
+After successfully executing you'll notice the created file structure in the ```modules/addressbook``` directory:
```

Comment:
When you quote just a word like modules/addressbook then just use a singel \` instead of 3 \`

(github just destroyed my example in the previous comment :-)